src/screens/App/components/index.js

```diff
@@ -1,3 +1,4 @@
+export { default } from './AppEntry'
```

Comment:
You probably don't want to do this - it's ok for modules but causes issues when tree-shaking the size of your javascript bundle for an app. In fact it was one of the reasons I was suggesting we move away from this folder structure